### DISCUSSION TITLE
CacheLocation overrides

### DISCUSSION THREAD

**Discussion by jochenberger** [equodev/equo-ide#124] [Q&A] [answered]
The CacheLocations class claims that cache directories can be overridden using `gradle.properties`. https://github.com/equodev/equo-ide/blob/main/solstice/src/main/java/dev/equo/solstice/p2/CacheLocations.java#L35-L45
How does this mechanism work? I couldn't manage to make it use different directories.
I'm asking because I'm trying to find a workaround for https://github.com/diffplug/spotless/issues/1687.

**Comment by jochenberger** [comment]
@nedtwigg, could you please help me out or point me to some docs that explain how this is supposed to work?

**Reply by nedtwigg** [reply]
Sorry for the delay Jochen, I'm working on a PR for this right now to move the cache directories to better defaults.

**Reply by nedtwigg** [reply, marked as answer]
FYI, the PR in question is

- https://github.com/equodev/equo-ide/pull/131

And it has been released in `solstice 1.3.1`, `spotless-plugin-gradle 6.19.0`, and `spotless-plugin-maven 2.37.0`.
